### DISCUSSION TITLE
Process for packaging

### DISCUSSION THREAD

**Discussion by SebastianStehle** [BuilderIO/jsx-lite#975] [Q&A] [answered]
Really cool project. Do you have a process to create npm packages and sample applications?

**Comment by samijaber** [comment, marked as answer]
@SebastianStehle Not currently. Right now, it's up to the user to do so manually. We are still exploring how users build npm packages out of Mitosis-generated code. 

Once we have enough data points and a solid understanding of user needs, we might try to add tooling around this.

**Reply by SebastianStehle** [reply]
Thanks for your answer.
